### ISSUE TITLE
Order-by actor transforms origin multiple times

### ISSUE THREAD

**Issue by rubensworks** [comunica/comunica#434]
#### Issue type:

- :bug: Bug

____
#### Description:

When executing a [SPARQL spec test](http://w3c.github.io/rdf-tests/sparql11/data-sparql11/json-res/manifest.ttl#jsonres01), I get the following error:

```
Error: The source already has a destination
    at SimpleTransformIterator.TransformIterator._validateSource (/Users/rtaelman/Documents/UGent/nosync/comunica/node_modules/asynciterator/asynciterator.js:1019:11)
    at SimpleTransformIterator.set [as source] (/Users/rtaelman/Documents/UGent/nosync/comunica/node_modules/asynciterator/asynciterator.js:984:10)
    at SimpleTransformIterator.TransformIterator (/Users/rtaelman/Documents/UGent/nosync/comunica/node_modules/asynciterator/asynciterator.js:969:27)
    at new SimpleTransformIterator (/Users/rtaelman/Documents/UGent/nosync/comunica/node_modules/asynciterator/asynciterator.js:1135:21)
    at PromiseProxyIterator.AsyncIterator.transform (/Users/rtaelman/Documents/UGent/nosync/comunica/node_modules/asynciterator/asynciterator.js:1264:10)
    at ActorQueryOperationOrderBySparqlee.runOperation (/Users/rtaelman/Documents/UGent/nosync/comunica/packages/actor-query-operation-orderby-sparqlee/lib/ActorQueryOperationOrderBySparqlee.js:48:46)
```

This is because AsyncIterator does not allow `transform` to be called multiple times on a stream.

**Comment by wschella** [comment]
Is the spec test something that was added after we started using OrderBySparqlee? If so, could you check whether this also happens with OrderByDirect? Also, do you know which test case fails? I'm assuming one of the 2 ASKS, since the other ones don't use OrderBy.

**Comment by rubensworks** [comment]
This is the test it fails for: http://w3c.github.io/rdf-tests/sparql11/data-sparql11/json-res/manifest.ttl#jsonres01 (also linked in the OP)

**Comment by wschella** [comment]
Should be closed by #436 (although i still don't quite understand why orderby get's triggered for that query)

**Comment by rubensworks** [comment]
Because that query does have an order-by :-)

```sparql
PREFIX : <http://example.org/>

SELECT * WHERE { ?s ?p ?o} ORDER BY ?s ?p ?o
```
